### ISSUE TITLE
.dmg for macOS from latest release (v.1.2.1) is not working (El Cap and older)

### ISSUE THREAD

**Issue by m2habert** [qutebrowser/qutebrowser#3722]
It just doesn't open.

(macOS 10.11.6)

**Comment by The-Compiler** [comment]
What happens if you run `/Applications/qutebrowser.app/Contents/MacOS/qutebrowser` in a terminal?

**Comment by m2habert** [comment]
Following error message:

`[10686] Error loading Python lib '/Users/m2habert/Desktop/qutebrowser.app/Contents/MacOS/.Python': dlopen: dlopen(/Users/m2habert/Desktop/qutebrowser.app/Contents/MacOS/.Python, 10): Symbol not found: _clock_getres
  Referenced from: /Users/m2habert/Desktop/qutebrowser.app/Contents/MacOS/.Python (which was built for Mac OS X 10.13)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/m2habert/Desktop/qutebrowser.app/Contents/MacOS/.Python
`

**Comment by The-Compiler** [comment]
Argh. That was built on macOS High Sierra - looks like it won't support older macOS versions that way :worried:

**Comment by m2habert** [comment]
Here hoping it's an easy fix... ¿Shoul I update the issue title?

**Comment by The-Compiler** [comment]
The fix will probably be rebuilding it on an older macOS, which is currently not something I can do easily.

**Comment by m2habert** [comment]
(I probably should have noted that v.1.1.1 is opening fine in OS 10.11.6)

**Comment by Linuus** [comment]
I have the same issue on macOS Sierra. I'll downgrade to 1.1.1 again :)

**Comment by The-Compiler** [comment]
I just released v1.3.0 which was built on Sierra (rather than High Sierra), which seemed to work down to El Capitan before. Assuming it's fine to close this, let me know if you still have problems though!

As an aside: Thanks to [MacStadium](https://www.macstadium.com/) for making this possible via their [Opensource Initiative](https://www.macstadium.com/opensource/) :+1:

**Comment by ynakao** [comment]
It seems still exist the isssue. v1.3.0 can't be opened by both clicking the icon and typing command from terminal. Error output is 

```
[nakaoyuji@UG-MBP.local][~]% /Applications/qutebrowser.app/Contents/MacOS/qutebrowser
[624] Error loading Python lib '/Applications/qutebrowser.app/Contents/MacOS/.Python': dlopen: dlopen(/Applications/qutebrowser.app/Contents/MacOS/.Python, 10): Symbol not found: _clock_getres
  Referenced from: /Applications/qutebrowser.app/Contents/MacOS/.Python (which was built for Mac OS X 10.12)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Applications/qutebrowser.app/Contents/MacOS/.Python
```

My environment is El Capitan v10.11.6.

**Comment by m2habert** [comment]
When searching "libSystem.B.dylib" "dynamic" "_clock_getres" I found the following issue:

https://github.com/Homebrew/homebrew-core/issues/3727

Seems relevant (it's way over my head) : 

> See # 2674 for some background, but the gist of it is this:
> 
> When building on OS X 10.11 with Xcode 8 against the macOS 10.12 SDK, you'll see the symbols added in macOS 10.12 e.g. clock_gettime and getentropy. This will cause many autoconf using projects to enable code that uses them. The built products will then fail with dynamic linker errors at runtime because the symbols aren't available on the 10.11 host system.

**Comment by The-Compiler** [comment]
It looks like I need to downgrade to El Capitan (10.11) and build the releases there. Interestingly, the older releases (e.g. v1.2.0) were built on Sierra too, but on an older Mac Mini, so maybe there was some legacy fallback or something active there which made it work...

**Comment by The-Compiler** [comment]
Oh, also, let me note that I won't be supporting El Capitan forever. As soon as Qt drops official support for it (which is probably whatever Qt release follows after macOS 10.14 is released), I'll also only support macOS Sierra (10.12) or newer.

**Comment by The-Compiler** [comment]
Can someone on El Capitan please test whether [qutebrowser-1.3.0-qt510-elcap.dmg](https://qutebrowser.org/tmp/qutebrowser-1.3.0-qt510-elcap.dmg) works?

**Comment by jgkamat** [comment]
I think that after qt drops support you may be able to run qb with older qt versions, but provided releases are usually built with the latest release. I would guess if you pinned your pyqt version you probably could use the latest build assuming we still support that old qt version.

**Comment by m2habert** [comment]
@The-Compiler  qB seems to be working great (also v10.11.6) (woooooooo-hooo, carrot mode!)

**Comment by The-Compiler** [comment]
I've released v1.3.1 and v1.3.2 in the meantime, which are built in the same system - so this should hopefully be fixed!
